scheduled-jobs/build/bug-sweeps/sweep/Jenkinsfile

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env groovy
+
+node {
+    checkout scm
+    def buildlib = load("pipeline-scripts/buildlib.groovy")
+    def commonlib = buildlib.commonlib
+
+    properties(
+        [
+            buildDiscarder(
+                logRotator(
+                    artifactDaysToKeepStr: '',
+                    artifactNumToKeepStr: '',
+                    daysToKeepStr: '',
+                    numToKeepStr: '90'
+                )
+            ),
+            [
+                $class : 'ParametersDefinitionProperty',
+                parameterDefinitions: [
+                    commonlib.ocpVersionParam('BUILD_VERSION'),
```

Comment:
Not specifying a `majorVersion` parameter here so we will default to `all` which leaves this open to easily add more per-version sweep jobs later.

Reply:
Not sure I follow here. Don't we restrict to the version we just built?

Reply:
There are 2 jobs in this PR

* `sweep` which is the parameterized job. It's like running `ocp` directly (which is the `Jenkinsfile` you commented on)
* `4.1` - Calls the `sweep` job with `BUILD_VERSION` set to `4.1`

> Don't we restrict to the version we just built?

Since we still do `n-2` advisories I could see us adding `3.11` and `3.10` and `3.9` jobs in here that would call the `sweep` job w/ the `BUILD_VERSION` set appropriately.